.github/workflows/update-deps.yml

```diff
@@ -0,0 +1,37 @@
+name: Update Dependencies
+
+on:
+  # Run every day.
+  schedule:
+    - cron: '0 3 * * *'
+  # And on on every PR merge so we get the updated dependencies ASAP, and to make sure the changelog doesn't conflict.
+  push:
+    branches:
+      - master
+
+jobs:
+  update-cli:
+    uses: getsentry/github-workflows/.github/workflows/updater.yml@v1
+    with:
+      path: script/sentry-cli.properties
+      name: CLI
+    secrets:
+      api_token: ${{ secrets.CI_DEPLOY_KEY }}
+  update-binaries:
+    runs-on: ubuntu-latest
+    needs: update-cli
+    steps:
+      - name: Checkout PR branch
+        if: needs.update-cli.outputs.originalTag != needs.update-cli.outputs.latestTag
```

Comment:
just wondering - can this condition be moved to the job? I've never used it that way but seems to be possible: https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution

Reply:
Yeah sounds good, less duplication. 👍